internal/cmd/schema-registry/command_cluster_update.go

```diff
@@ -73,7 +77,50 @@ func (c *command) updateTopLevelCompatibility(cmd *cobra.Command) error {
 		return err
 	}
 
-	updateReq := srsdk.ConfigUpdateRequest{Compatibility: strings.ToUpper(compatibility)}
+	compatibilityGroup, err := cmd.Flags().GetString("compatibility-group")
+	if err != nil {
+		return err
+	}
+
+	var defaultMetadata srsdk.Metadata
+	var overrideMetadata srsdk.Metadata
+	var defaultRuleset srsdk.RuleSet
+	var overrideRuleset srsdk.RuleSet
+
+	metadataDefaultsPath, err := cmd.Flags().GetString("metadata-defaults")
+	if err != nil {
+		return err
+	}
+	rulesetDefaultPath, err := cmd.Flags().GetString("ruleset-defaults")
+	if err != nil {
+		return err
+	}
+	err = readMetadataAndRuleset(metadataDefaultsPath, &defaultMetadata, rulesetDefaultPath, &defaultRuleset)
```

Comment:
How about returning the metadata and ruleset instead of passing by pointer? The former is much more explicit.

Reply:
In your previous comment you suggest passing by pointers https://github.com/confluentinc/cli/pull/1837#discussion_r1160022691 
Which way is preferred? Asking since I'm not an expert in golang.

Reply:
Oops, I forgot about that (it's been more than a month), disregard this comment!